paasta_itests/steps/setup_marathon_job_steps.py

```diff
@@ -93,6 +96,18 @@ def create_complete_app(context):
         assert 'deployed' in return_tuple[1]
 
 
+@when(u'we change the number of instances to {number}')
+def change_number_of_context_instances(context, number):
+    fake_service_config['instances'] = int(number)
+
+
+@when(u'we run setup_marathon_job for {number}s')
+def run_setup_marathon_job(context, number):
+    for _ in xrange(int(number)):
+        create_complete_app(context)
```

Comment:
This isn't obvious to me how many instances we are starting off with. can you make *that* the parameter? "we run setup_marathon_job on a service with an instance count of X"